package.json

```diff
@@ -42,6 +42,8 @@
     "grunt-cli": "latest",
     "grunt-cucumber": "^0.3.0",
     "mocha": "^2.4.5",
+    "mysql": "^2.13.0",
```

Comment:
Once this is merged the repo scanner will see this dependency and automatically configure the job to run in a MySQL environment.